murenn/dtcwt/nn.py

```diff
@@ -88,52 +86,66 @@ def to_conv1d(self):
         Compute the single-resolution equivalent impulse response of the MuReNN layer.
         This would be helpful for visualization in Fourier domain, for receptive fields,
         and for comparing computational costs.
-           DTCWT        conv1d        IDTCWT
-        δ -------> ψ_j --------> w_jq -------> y_jq
+           conv1d        IDTCWT
+        δ --------> w_jq -------> y_jq
```

Comment:
i don't think this diagram is correct either. we don't start from a dirac in the time domain anymore. instead we build a dirac in the DTCWT domain, on either tree (a and b)

Reply:
how should I describe this operation? use $\delta_j$ instead of $\delta$?

Reply:
re-reading the diagram, i now think it's actually good!